### PR TITLE
[FIX] Python3 conversion bug fixes (Round 2)

### PR DESCRIPTION
**Pull request by DESm1th**

### CODE REVIEW

File: bin/dm_qc_report.py [hunk 1]
```diff
@@ -504,8 +504,8 @@ def get_series_to_add(series, subject):
         t2 = get_split_image(subject, series.series_num, 'T2')
```
Comment (josephmje) [inline bin/dm_qc_report.py]: I think we can do away with this bit. `dm_xnat_extract.py` now takes care of splitting the PDT2 scan into separate files.

**Comment by jerdra** [comment]
looks good to me!

File: bin/dm_qc_report.py (hunk 1 above)
Comment (jerdra) [reply bin/dm_qc_report.py]: might be out of scope of PR, could maybe do a separate one for this specific change?

Comment (josephmje) [reply bin/dm_qc_report.py]: yup i'm fine with that.

Comment (DESm1th) [reply bin/dm_qc_report.py]: Ooh good catch. I'll roll that into another one soon (if someone doesnt beat me to it)

**Pull request merged**